Makefile

```diff
@@ -1,2 +1,5 @@
 test:
 	poetry run pytest
+
```

Comment:
while you're at it: maybe also `makemigrations` for `python manage.py makemigrations api`?

Reply:
Done